components/SearchResult.vue

```diff
@@ -1,18 +1,19 @@
 <template>
     <div class="search-result">
-        <div class="result-header flex">
-            <img alt="profile icon" class="profile-icon" src="@/assets/images/profileIcon.svg" width="25"
-                height="25" />
-            <span class="flex-1 w-4 font-bold pl-2"> {{ name }} </span>
+        <div class="result-header ml-4">
+            <svg role="img" alt="profile icon" title="profile icon" class="profile-icon w-8 h-8 stroke-primary inline">
+                <use xlink:href="../assets/images/profile-icon.svg#profile-icon-svg" class="w-[20px] h-[20px]" />
+            </svg>
+            <span class="flex-1 w-4 font-bold pl-2 self-center"> {{ name }} </span>
```

Comment:
Just curious, why `flex-1` here?

Reply:
Don't need it anymore. I changed from flex to inline block. 
I reaping the cost of not reviewing my css changes 😂

Reply:
Good idea. Can you make an issue for it? Or shall I?(I can't remember what we decided on this question)